testing/trino-testing/src/main/java/io/trino/testing/AbstractTestQueries.java

```diff
@@ -165,8 +165,8 @@ public void testLimit()
     @Test
     public void testLimitWithAggregation()
     {
-        MaterializedResult actual = computeActual("SELECT custkey, SUM(CAST(totalprice * 100 AS BIGINT)) FROM orders GROUP BY custkey LIMIT 10");
-        MaterializedResult all = computeExpected("SELECT custkey, SUM(CAST(totalprice * 100 AS BIGINT)) FROM orders GROUP BY custkey", actual.getTypes());
+        MaterializedResult actual = computeActual("SELECT custkey, SUM(orderkey) FROM orders GROUP BY custkey LIMIT 10");
```

Comment:
as long there is no casting of a double involved, it should works good. 
thanks @losipiuk